### ISSUE TITLE
Fix Bug with Inventory addition page

### ISSUE THREAD

**Issue by ashakhatri007** [ashakhatri007/CSC510_Group25_Project1#43]
While playing with add inventory form on my local, I noticed that inventory gets added only once after that the latest will remain and rest are deleted.
After investigation found that the id is harcoded and hence database is adding only latest entry by replacing the entry matching id.
We need to fic this

**Comment by divyankgupta23** [comment]
Also while testing it I found that the inventory is not getting refreshed when an item is added. You can make a new issue for this feature.

**Comment by ashakhatri007** [comment]
@divyankgupta23 , sure! I think @hetthakkar was going to deal with it, created new issue to track this.
